examples/reference.yml

```diff
@@ -11,9 +11,12 @@
 #      rootcertbundle: "/path/to/server.pem"
 
 server:  # Server settings.
-  # Address to listen on.
+  # Address to listen on. Can be HOST:PORT for TCP or file path for Unix socket.
```

Comment:
Could you add an example of a filepath here? (eg.`/run/docker_auth.sock` or whichever is the correct format)?